resources/views/layouts/app.blade.php

```diff
@@ -47,6 +49,10 @@
                         </div>
                     </li>
 
+                    <li class="navbar-nav me-auto">
```

Comment:
class here should be nav-item and add class to anchor 'nav-link' to show the item properly

Reply:
Done